diff -ru curl-7.50.3/winbuild/MakefileBuild.vc curl-7.50.3.patch/winbuild/MakefileBuild.vc
--- curl-7.50.3/winbuild/MakefileBuild.vc	2016-10-14 18:53:09.211275300 +0200
+++ curl-7.50.3.patch/winbuild/MakefileBuild.vc	2016-10-14 18:53:03.914702500 +0200
@@ -56,11 +56,11 @@
 !IF "$(VC)"=="6"
 CC_NODEBUG  = $(CC) /O2 /DNDEBUG
 CC_DEBUG    = $(CC) /Od /Gm /Zi /D_DEBUG /GZ
-CFLAGS      = /I. /I../lib /I../include /nologo /W4 /wd4127 /GX /DWIN32 /YX /FD /c /DBUILDING_LIBCURL
+CFLAGS      = /I. /I../lib /I../include /nologo /W4 /wd4127 /GX /DWIN32 /YX /FD /c /DBUILDING_LIBCURL /DUSE_SYNC_DNS
 !ELSE
 CC_NODEBUG  = $(CC) /O2 /DNDEBUG
 CC_DEBUG    = $(CC) /Od /D_DEBUG /RTC1 /Z7 /LDd
-CFLAGS      = /I. /I ../lib /I../include /nologo /W4 /wd4127 /EHsc /DWIN32 /FD /c /DBUILDING_LIBCURL
+CFLAGS      = /I. /I ../lib /I../include /nologo /W4 /wd4127 /EHsc /DWIN32 /FD /c /DBUILDING_LIBCURL /DUSE_SYNC_DNS
 !ENDIF
 
 LFLAGS     = /nologo /machine:$(MACHINE)
